SignalMessaging/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -124,6 +224,77 @@ public class AttachmentApprovalViewController: UIPageViewController, UIPageViewC
 
     // MARK: - View Helpers
 
+    func buildRemoveButton(attachmentItem: SignalAttachmentItem) -> RemoveAttachmentButton {
+        let button = RemoveAttachmentButton()
+        button.setImage(#imageLiteral(resourceName: "ic_small_x"), for: .normal)
+        button.attachmentItem = attachmentItem
+        return button
+    }
```

Comment:
Should we write a (cliché) UIButton subclass (`OWSButton`?) that is block-based to avoid the need for this kind of subclass?